tests/test_utils_docs.py

```diff
@@ -140,7 +140,7 @@ def mk_sample(v):
         )
     }, dataset_search_fields={})
 
-    assert without_lineage_sources(mk_sample(10), no_sources_type) == mk_sample(10)
+    assert without_lineage_sources(mk_sample(10), no_sources_type)["lineage"] == {}
```

Comment:
I don't think this test covers the case where `lineage` doesn't contain `source_datasets`?